### PR TITLE
Add support for FASTA tileset

### PR DESCRIPTION
**Pull request by 101arrowz**
## Description

What was changed in this pull request?

Adds support for FASTA tileset
Depends on https://github.com/higlass/clodius/pull/138

Why is it necessary?

Useful for tracks that need access to sequence data

## Checklist

- [ ] <strike>Unit tests added or updated</strike> N/A
- [x] Documentation added or updated
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: tilesets/views.py [hunk 1]
```diff
@@ -648,6 +651,16 @@ def tileset_info(request):
...
+        elif tileset_object.filetype == 'fasta':
+            chromsizes = tgt.get_chromsizes(tileset_object)
+            tsinfo = hgfa.tileset_info(
+                    tileset_object.datafile.path,
+                    chromsizes
+                )
+            #print('tsinfo:', tsinfo)
+            if 'chromsizes' in tsinfo:
+                tsinfo['chromsizes'] = [(c, int(s)) for c,s in tsinfo['chromsizes']]
+            tileset_infos[tileset_uuid] = tsinfo
```
Comment (pkerpedjiev) [inline tilesets/views.py:663]: See comment in the clodius PR about potentially retrieving too much data. Here's how it's handled in BAM files for the tileset_info: https://github.com/higlass/higlass-server/blob/develop/tilesets/views.py#L699

File: tilesets/generate_tiles.py [hunk 2]
```diff
@@ -529,6 +530,9 @@ def generate_tiles(tileset_tile_ids):
     elif tileset.filetype == 'bigwig':
         chromsizes = get_chromsizes(tileset)
         return hgbi.tiles(tileset.datafile.path, tile_ids, chromsizes=chromsizes)
+    elif tileset.filetype == 'fasta':
+        chromsizes = get_chromsizes(tileset)
+        return hgfa.tiles(tileset.datafile.path, tile_ids, chromsizes=chromsizes)
```
Comment (pkerpedjiev) [inline tilesets/generate_tiles.py:535]: See comment about retrieving too much data in the clodius PR. Here's how it's handled for BAM tiles: https://github.com/higlass/higlass-server/blob/develop/tilesets/generate_tiles.py#L548

**Review by pkerpedjiev — COMMENTED** [review]
Looks good. Just a couple comments to go with the ones in the clodius PR.

**Comment by 101arrowz** [comment]
I've added the maximum tile width and a corresponding environment variable to configure it. Not sure there's a need for a unit test since there will be one within Clodius and it's following the exact same structure as the other tilesets.

**Comment by pkerpedjiev** [comment]
Cool, thank you! Yeah, no need for a unit test if it's covered by clodius.

**Pull request merged**